vendor/github.com/newrelic/infra-integrations-sdk/jmx/jmx.go

```diff
@@ -63,21 +73,25 @@ func (cfg *connectionConfig) command() []string {
 	if os.Getenv("NR_JMX_TOOL") != "" {
 		c = strings.Split(os.Getenv("NR_JMX_TOOL"), " ")
 	} else {
-		c = []string{jmxCommand}
+		c = []string{cfg.executablePath}
 	}
 
-	c = append(c, "--hostname", cfg.hostname, "--port", cfg.port)
-	if cfg.uriPath != "" {
-		c = append(c, "--uriPath", cfg.uriPath)
-	}
-	if cfg.username != "" && cfg.password != "" {
-		c = append(c, "--username", cfg.username, "--password", cfg.password)
-	}
-	if cfg.remote {
-		c = append(c, "--remote")
-	}
-	if cfg.remoteJBossStandalone {
-		c = append(c, "--remoteJBossStandalone")
+	if cfg.connectionURL != "" {
+		c = append(c, "--connURL", cfg.connectionURL)
```

Comment:
is it expected that the connection URL contains the username and password now?

Reply:
nope, this is just the connection URL as explained in Java docs

Reply:
Sorry -- I'm a little confused. Then why do we only add username and password to the `nrjmx` command if connectionURL is unset?

Reply:
Ops my mistake!

Reply:
Done https://github.com/newrelic/infra-integrations-sdk/pull/199

Reply:
Restored at https://github.com/newrelic/nri-jmx/pull/47/commits/c698d75407d10c44202daaa454929d944baf21c6